content/articles/letsencrypt.md

```diff
@@ -58,7 +58,7 @@ Some Let's Encrypt features may not be supported by DNSimple. Check the [limitat
 The DNSimple Let's Encrypt integration allows you to request an SSL certificate for free using the Let's Encrypt certification authority.
 
 <note>
-To request an SSL certificate with Let's Encrypt, the domains **must be delegated and resolving with DNSimple**. The domain doesn't need to be registered with DNSimple.
+To request an SSL certificate with Let's Encrypt, the domains **must be delegated and resolving only with DNSimple**. The domain doesn't need to be registered with DNSimple. This means it is not compatible with Secondary DNS.
```

Comment:
```suggestion
To request an SSL certificate with Let's Encrypt, the domains **must be delegated and resolved exclusively with DNSimple**. The domain doesn't need to be registered with DNSimple, which means it is not compatible with Secondary DNS.
```

Reply:
I merged this with the other suggestions, thanks!